File: src/main/scala/emasher/sockets/CommonProxy.java
package emasher.sockets;

import cpw.mods.fml.relauncher.Side;
import emasher.sockets.packethandling.*;

public class CommonProxy {
	public void registerRenderers() {
		// Nothing here as this is the server side proxy
	}

	public void registerMessages() {
		SocketsMod.network.registerMessage( RequestInfoFromServerMessage.Handler.class, RequestInfoFromServerMessage.class, 6, Side.SERVER );

		SocketsMod.network.registerMessage( SocketStateMessage.Handler.class, SocketStateMessage.class, 0, Side.CLIENT );
		SocketsMod.network.registerMessage( SocketItemMessage.Handler.class, SocketItemMessage.class, 1, Side.CLIENT );
		SocketsMod.network.registerMessage( SocketFluidMessage.Handler.class, SocketFluidMessage.class, 2, Side.CLIENT );
		SocketsMod.network.registerMessage( PipeColourMessage.Handler.class, PipeColourMessage.class, 3, Side.CLIENT );
		SocketsMod.network.registerMessage( AdapterSideMessage.Handler.class, AdapterSideMessage.class, 4, Side.CLIENT );
		SocketsMod.network.registerMessage( ChangerSideMessage.Handler.class, ChangerSideMessage.class, 5, Side.CLIENT );
	}
}

File: src/main/scala/emasher/CommonProxy.scala
package emasher

import cpw.mods.fml.relauncher.Side
import emasher.packethandling._

class CommonProxy {
  def registerMessages(): Unit = {
    EngineersToolbox.network.registerMessage( classOf[RequestInfoFromServerMessage.Handler], classOf[RequestInfoFromServerMessage], 6, Side.SERVER )
    EngineersToolbox.network.registerMessage( classOf[SocketStateMessage.Handler], classOf[SocketStateMessage], 0, Side.CLIENT )
    EngineersToolbox.network.registerMessage( classOf[SocketItemMessage.Handler], classOf[SocketItemMessage], 1, Side.CLIENT )
    EngineersToolbox.network.registerMessage( classOf[SocketFluidMessage.Handler], classOf[SocketFluidMessage], 2, Side.CLIENT )
    EngineersToolbox.network.registerMessage( classOf[PipeColourMessage.Handler], classOf[PipeColourMessage], 3, Side.CLIENT )
    EngineersToolbox.network.registerMessage( classOf[AdapterSideMessage.Handler], classOf[AdapterSideMessage], 4, Side.CLIENT )
    EngineersToolbox.network.registerMessage( classOf[ChangerSideMessage.Handler], classOf[ChangerSideMessage], 5, Side.CLIENT )
  }

  def registerRenderers(): Unit = {

  }

  def registerNEI(): Unit = {

  }
}
